org.eclim.jdt/java/org/eclim/plugin/jdt/command/complete/CodeCompleteCommand.java

```diff
@@ -185,6 +184,9 @@ protected CodeCompleteResult createCompletionResult(
     // of whether the user ever views it.
     /*return new CodeCompleteResult(
         kind, completion, menu, proposal.getAdditionalProposalInfo());*/
-    return new CodeCompleteResult(completion, menu, menu, type, offset);
+    CodeCompleteResult result = new CodeCompleteResult(completion, menu, menu, type, offset);
+    result.setRelevance(proposal.getRelevance());
```

Comment:
Can you provide an example? I'm not saying that there shouldn't be an option to sort by relevance, but it should be just that, and option.